config/locales/find.yml

```diff
@@ -149,27 +154,21 @@ en:
       skilled_worker_visa:
         not_available:
           html:
-            <p class="govuk-body">If you do not already have the right to work in the UK, you may need to apply for a visa. The main visa for salaried courses is the <a class="govuk-link" href="https://www.gov.uk/skilled-worker-visa">Skilled Worker visa.</a></p>
-            <p class="govuk-body">Sponsorship for a Skilled Worker visa is not available for this course.</p>
-            <p class="govuk-body">If you need a visa, filter your course search to find courses with visa sponsorship.</p>
-            <p class="govuk-body">You can also <a class="govuk-link" href="https://getintoteaching.education.gov.uk/non-uk-teachers/visas-for-non-uk-trainees">learn more about different types of visa</a> which allow you to train to be a teacher without being sponsored.</p>
+            <p class="govuk-body">If you do not already have the right to study in the UK, you may need to <a class="govuk-link" href="https://getintoteaching.education.gov.uk/non-uk-teachers/visas-for-non-uk-trainees"> apply for your visa to train to teach in England </a>.</p>
```

Comment:
I think these spaces are causing a bit of funky formatting? 🤔 

<img width="966" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/4a27d8d9-f27c-4a2b-ac8f-f6aef71a206e">

Reply:
Good catch